app_builder/lib/app_builder.ex

```diff
@@ -1,5 +1,213 @@
+  def copy_elixir(release) do
+    include_executables_for = Keyword.get(release.options, :include_executables_for, [:unix])
+
+    elixir_bin_path = Application.app_dir(:elixir, "../../bin")
+    bin_path = Path.join(release.path, "bin")
+    File.mkdir_p!(bin_path)
+
+    # download and unzip
+    standalone_destination = Path.join(release.path, "elixir")
+    download_elixir_at_destination(standalone_destination)
+
+    # patch elixir file to look for the right erts <resource_path>/rel/releases/#{release.version}/elixir
+    patch_elixir(include_executables_for, release,
+      fn filename ->
+        Path.join(elixir_bin_path, filename)
+      end,
+      fn filename ->
+        Path.join(release.version_path, filename)
+      end
+    )
+
+    # patch elixir file to look for the right erts <resource_path>/rel/elixir/bin/elixir
+    patch_elixir(include_executables_for, release,
+      fn filename ->
+        Path.join([standalone_destination, "bin", filename])
+      end,
+      fn filename ->
+        Path.join([standalone_destination, "bin", filename])
+      end
+    )
+
+    release
+  end
+
+  @elixir_standalone_version "1.13.2"
+
+  defp download_elixir_at_destination(destination) do
+    url = "https://github.com/elixir-lang/elixir/releases/download/v#{@elixir_standalone_version}/Precompiled.zip"
+    binary = fetch_body!(url)
+    File.write!("/tmp/elixir_#{@elixir_standalone_version}.zip", binary, [:binary])
+    :zip.unzip('/tmp/elixir_#{@elixir_standalone_version}.zip', cwd: destination)
+  end
+
+  defp patch_elixir(include_executables_for, release, fn_source, fn_target) do
+    for os <- include_executables_for do
+      for {filename, contents_fun} <- elixir_cli_for(os, release) do
+        source = fn_source.(filename)
+
+        if File.regular?(source) do
+          target = fn_target.(filename)
+
+          File.write!(target, contents_fun.(source))
+          executable!(target)
+
+        else
+          skipping("#{filename} for #{os} (bin/#{filename} not found in the Elixir installation)")
+        end
+      end
+    end
+  end
+
+  defp elixir_cli_for(:unix, release) do
+    [
+      {"elixir",
+       &(&1
+         |> File.read!()
+         |> String.replace(~s[ -pa "$SCRIPT_PATH"/../lib/*/ebin], "")
+         |> replace_erts_bin(release, ~s["$SCRIPT_PATH"/../../erts-#{release.erts_version}/bin/]))},
+      {"iex", &File.read!/1}
+    ]
+  end
+
+  defp elixir_cli_for(:windows, release) do
+    [
+      {"elixir.bat",
+       &(&1
+         |> File.read!()
+         |> String.replace(~s[goto expand_erl_libs], ~s[goto run])
+         |> replace_erts_bin(release, ~s[%~dp0\\..\\..\\erts-#{release.erts_version}\\bin\\]))},
+      {"iex.bat", &File.read!/1}
+    ]
+  end
+
+  defp erts_data do
+    version = :erlang.system_info(:version)
+    {:filename.join(:code.root_dir(), 'erts-#{version}'), :filename.join(:code.root_dir(), 'bin'), :code.lib_dir(), version}
+  end
+
+  defp fetch_body!(url) do
```

Comment:
There is already `Livebook.Utils.HTTP.request`, maybe we can use it? 🤔

Reply:
`Livebook.Utils.HTTP.request`, It causes me timeouts; I set it to 15_000 to reduce them.